GPU/Common/ReplacedTexture.cpp

```diff
@@ -185,7 +235,63 @@ bool ReplacedTexture::LoadLevelData(ReplacedTextureLevel &level, int mipLevel) {
 	std::string magic;
 	ReplacedImageType imageType = Identify(vfs_, openFile, &magic);
 
-	if (imageType == ReplacedImageType::ZIM) {
+	int ddsBytesToRead = 0;  // Used by the DDS reader only.
+	bool ddsDX10 = false;
+	int numMips = 1;
+
+	if (imageType == ReplacedImageType::DDS) {
+		DDSHeader header;
+		DDSHeaderDXT10 header10{};
+		good = vfs_->Read(openFile, &header, sizeof(header)) == sizeof(header);
+
+		*pixelFormat = Draw::DataFormat::UNDEFINED;
+		u32 format;
+		if (good && (header.ddspf.dwFlags & DDPF_FOURCC)) {
+			char *fcc = (char *)&header.ddspf.dwFourCC;
+			INFO_LOG(G3D, "DDS fourcc: %c%c%c%c", fcc[0], fcc[1], fcc[2], fcc[3]);
```

Comment:
Seems a bit chatty for a large texture pack...

-[Unknown]

Reply:
Yeah got rid of it later.